docs/reference/mapping/params/format.asciidoc

```diff
@@ -66,8 +66,9 @@ The following tables lists all the defaults ISO formats supported:
 
 [[strict-date-time]]`date_optional_time` or `strict_date_optional_time`::
 
-    A generic ISO datetime parser where the date is mandatory and the time is
-    optional.
+    A generic ISO datetime parser where the date, in `year_month_day format`, is
+    mandatory and the time, separaetd by `T`, is optional.
```

Comment:
```suggestion
    mandatory and the time is optional.
```

Based on my tests, including a `T` in the input will result in an error. However, the following format worked:

```
[source,console]
--------------------------------------------------
PUT my_index
{
  "mappings": {
    "properties": {
      "date": {
        "type":   "date",
        "format": "strict_date_optional_time || yyyy-MM-dd hh:mm:ss.SSSZ"
      }
    }
  }
}
--------------------------------------------------
```

Also: `separated` is misspelled.

Reply:
@jrodewig The `T` is definitely required, when a time is specified. What was the date you tested with?

Reply:
Hi @rjernst,

You're right. The `T` is required for the index API. My initial attempts were using the create index API, but after reviewing the docs, I think I just misinterpreted what was there.

My apologies @arkel-s . I'll clean up my review suggestions here.